doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -202,7 +202,12 @@ nRF5340 Audio
 nRF Desktop
 -----------
 
-|no_changes_yet_note|
+* Added Bluetooth LE Low Latency Packet Mode (LLPM) dependency (:kconfig:option:`CONFIG_CAF_BLE_USE_LLPM`) to :ref:`nrf_desktop_hfclk_lock`.
```

Comment:
Remove the changelog entries from this PR and instead, add them as suggestion to @divipillai's https://github.com/nrfconnect/sdk-nrf/pull/18192.

Reply:
Will do right after review

Reply:
Removed release notes